src/x509.c

```diff
@@ -4118,8 +4118,9 @@ static WOLFSSL_DIST_POINT_NAME* wolfSSL_DIST_POINT_NAME_new(void)
     }
     gns->type = STACK_TYPE_GEN_NAME;
 
+    /* Setting type to 0 because this is fullname */
```

Comment:
Can you add the RFC reference to this comment?

Reply:
RFC 5280 section 4.2.1.13, ASN1 definition of DistributionPointName. The fullName choice has type == 0.

Reply:
Thank you, we added it in the subsequent commit!
https://github.com/wolfSSL/wolfssl/pull/5721/commits/e409acc0f7582573318bee47cc7ac5edcebbfb31